File: python/agentic_commerce_protocol/models/checkout_session_create_request.py
# this file is @generated
import typing as t

from .address import Address
from .buyer import Buyer
from .common import BaseModel
from .item import Item


class CheckoutSessionCreateRequest(BaseModel):
    buyer: t.Optional[Buyer] = None
    items: t.List[Item]
    fulfillment_address: t.Optional[Address] = None

File: kotlin/src/main/kotlin/com/svix/agentic_commerce_protocol/CheckoutSessionCreateRequest.kt
// This file is @generated
package com.svix.agentic_commerce_protocol

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class CheckoutSessionCreateRequest(
    val buyer: Buyer? = null,
    val items: List<Item>,
    @SerialName("fulfillment_address") val fulfillmentAddress: Address? = null,
)
